src/pages/library/DataBrowserDetails.js

```diff
@@ -147,8 +147,9 @@ export const SidebarComponent = ({ dataObj, id }) => {
                   Ajax().Metrics.captureEvent(`${Events.catalogRequestAccess}:popUp`, {
                     // These are still using snapshot as a relic to ensure backwards search
                     // capabilities within the data browser.
-                    snapshotId: _.get('dct:identifier', dataObj),
-                    snapshotName: dataObj['dct:title']
+                    id: dataObj.id,
+                    // TODO: Make point at a data catalog title, rather than the underlying storage system
+                    title: dataObj['dct:title']
```

Comment:
I think this is correct. It's possible that `dct:title` is the same as a snapshot title, but it's not guaranteed to be so. When a data owner creates the catalog entry, they get to choose what `dct:title` contains. So I think `dct:title` is correct and don't need a TODO.

Reply:
What _is_ `dct:title`? My understanding was that it was the underlying data storage systems title, which I would expect us to not want to use, as opposed to something stored in our metadata (which might derive from `dct:title`?

Reply:
We don't yet have a specification for exactly what's in a catalog entry and what each attribute means, so there's no way you could know what this is. A `dct:title` and similarly named items are part of the TIM representation of a dataset, which is specified [here](https://github.com/DataBiosphere/terra-interoperability-model/blob/master/releases/1.x/terra-core/TerraDCAT-AP.ttl#L167). The attribute `dct:title` isn't directly specified on `TerraDCAT_ap:Dataset` but is inherited from `dct:Resource` (via `dcat:Dataset`):, see https://www.w3.org/TR/vocab-dcat-2/#Property:resource_title.

In theory the UI code should only be relying on properties from `TerraDCAT_ap:Dataset`, and any terms in that form are part of the catalog's unified representation and are not storage system specific.